wsgi/odyssy/events/views.py

```diff
@@ -0,0 +1,37 @@
+from django.shortcuts import get_object_or_404
```

Comment:
I have made such functions in views.py. But, it is better if such methods are defined in model itself. I will change it.